### PR TITLE
[_]:Feat/partners-LP

### PR DESCRIPTION
**Pull request by xabg2**
LP for partners finished with /partner-discount path.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,9 +13,12 @@
     "lint": "eslint './**/*.{js,ts,tsx}'"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
     "@headlessui/react": "1.6.6",
     "@iconscout/react-unicons": "^1.1.6",
     "@internxt/lib": "1.2.0",
+    "@mui/material": "^5.10.11",
```
Comment (sg-gs) [inline package.json:21]: why is `@mui/material` required? This library is big

**Review by sg-gs — APPROVED** [review]

**Comment by sg-gs** [comment]
Take into count that adding libraries is always an overhead which is going to be translated into bigger builds outputs which is going to affect things like load times of website @masterprog-cmd

File: package.json (hunk 1 above)
Comment (xabg2) [reply package.json:21]: Because I use Snackbar component from this library.

Comment (xabg2) [reply package.json:21]: Maybe I need  to find another package only with Snackbar component?

File: components/partner-discount/HeroSection.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+  const copyCoupon = () => {
+    if (isTechradar) {
+      navigator.clipboard.writeText(techradar);
+    } else {
+      navigator.clipboard.writeText(flash);
+    }
+    setOpen(true);
+  };
+
+  if (open === true) {
```
Comment (sg-gs) [inline components/partner-discount/HeroSection.js]: `if (open)` is enough

**Review by sg-gs — CHANGES_REQUESTED** [review]

**Review by sg-gs — APPROVED** [review]

**Pull request merged**